### PR TITLE
Make export() function public.

### PR DESCRIPTION
**Pull request by ChengYueJia**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,7 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
-riscv/runtime/target
\ No newline at end of file
+riscv/runtime/target
+
+.idea
+.vscode
```
Comment (leonardoalt) [inline .gitignore:15]: Can you revert the changes to this file

**Review by leonardoalt — COMMENTED** [review]
thanks! just a small comment

File: .gitignore (hunk 1 above)
Comment (ChengYueJia) [reply .gitignore:15]: ok

**Review by leonardoalt — APPROVED** [review]
